docs/examples.md

```diff
@@ -76,6 +76,24 @@ $y$j9T$A0Y3wwVOKP69S.1K/zYGN.$S596l11UGH3XjN...
 
 The `yescrypt` hashing method is recommended for new passwords. For more details on hashing methods, see `man 5 crypt`.
 
+In case inline definitions of SSH keys are undesirable you can use file references instead. This example creates three users (`core`, `jlebon`, `miabbott`) and sets up their respective SSH keys via file references instead of inlining them. This requires using `butane` with the `--files-dir` parameter to allow loading files from disk.
+
+### Handling SSH public keys
```

Comment:
"Handling SSH public keys" is a bit generic, since we've already introduced SSH key configuration.  Maybe "SSH keys from local files"?